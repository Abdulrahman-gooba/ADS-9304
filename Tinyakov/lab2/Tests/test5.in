(-(+ (*(r)(421))(*(-(2))(d)) )(/ (+(q)(+(w)(e)))(*(a)(*(b)(c))) )) a
