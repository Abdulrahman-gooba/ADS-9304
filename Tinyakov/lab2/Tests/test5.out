((((0)*(421)+(r)*(0)))+(((-(0))*(d)+(-(2))*(0))))-((((0)+((0)+(0)))*((a)*((b)*(c)))-((q)+((w)+(e)))*(((1)*((b)*(c))+(a)*(((0)*(c)+(b)*(0))))))/((a)*((b)*(c)))^2)
